src/App.tsx

```diff
@@ -2,108 +2,40 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+
+import usersFromServer from './api/users';
+import { PostFromServer, Post } from './types/Post';
+import { UserFromServer } from './types/User';
+import { CommentFromServer } from './types/Comment';
+import { PostList } from './components/PostList';
+
+export function getPostByID(
+  posts: PostFromServer[],
+  users: UserFromServer[],
+  commentsData: CommentFromServer[],
+): Post[] {
+  return posts.map((post) => ({
+    ...post,
+    user: users.find((user) => (
+      user.id === post.userId
+    )) || null,
+    comments: commentsData.filter((comment) => (
+      comment.postId === post.id
+    )) || null,
```

Comment:
```suggestion
    user,
    comments,
```
давай винесем це все в змінні